esd_services_api_client/beast/_connector.py

```diff
@@ -250,3 +254,17 @@ def start_job(self, job_params: BeastJobParams, job_name: str) -> Optional[str]:
             request_id, _ = self._submit(submit_request, job_name)
 
         return request_id
+
+    def read_configuration(
+        self, configuration_name: str
+    ) -> Optional[SparkSubmissionConfiguration]:
+        """
+          Returns a deployed SparkJob configuration.
+        :param configuration_name: Name of the configuration to find
+        :return: A SparkSubmissionConfiguration object, if found, or None
+        """
+        response = self.http.get(f"{self.base_url}/job/deployed/{configuration_name}")
+        if not response.ok:
```

Comment:
I think we should return None in case of the 404 status code, in other cases call  `raise_for_status`.